File: WorldFeed/Server/src/WorldFeed.Common/Models/Properties/TwitterExceptionInfo.cs
﻿namespace WorldFeed.Common.Models.Properties
{
    using Newtonsoft.Json;

    using global::WorldFeed.Common.Exceptions;

    public class TwitterExceptionInfo : ITwitterExceptionInfo
    {
        [JsonProperty("message")]
        public string Message { get; set; }

        [JsonProperty("code")]
        public int Code { get; set; }

        [JsonProperty("label")]
        public string Label { get; set; }
    }
}

File: Sharebook/Client/src/app/core/Core/Models/Properties/TwitterExceptionInfo.ts
import {Injectable} from "@angular/core";

import {ITwitterExceptionInfo} from "../../Exceptions/ITwitterExceptionInfo";

@Injectable({
  providedIn: 'root',
})
export class TwitterExceptionInfo implements ITwitterExceptionInfo {
  // [JsonProperty("message")];
  public message: string;

  // [JsonProperty("code")];
  public code: number;

  // [JsonProperty("label")];
  public label: string;
}
